### ISSUE TITLE
Codec priority

### ISSUE THREAD

**Issue by babayanta1994** [flutter-webrtc/dart-sip-ua#386]
We have a problem using the dart-sip-ua library. And as we found out, it is connected with the fact that the library uses the vp8 codec as a priority. How can we prioritize another video codec (eg h264)? Can you please help us with this?

**Comment by Madhav-Aman** [comment]
I am also facing the same issue>>!

**Comment by ygor-dextak** [comment]
Did any of you manage to resolve this issue?

**Comment by babayanta1994** [comment]
no, we are still waiting for a response from the developers. In the meantime, we are using another library and this one is on pause, since the developers do not want to support it

**Comment by tatsuyaueda** [comment]
Probably, but as mentioned in the next comment, I think that the codec is under the control of the webrtc side and cannot be controlled by dart-sip-ua.

https://github.com/flutter-webrtc/dart-sip-ua/issues/196#issuecomment-815010751
